Terminal.Gui/ConsoleDrivers/CursesDriver/ClipboardImpl.cs

```diff
@@ -168,33 +166,32 @@ private bool CheckSupport ()
     }
 
     [DllImport ("/System/Library/Frameworks/AppKit.framework/AppKit")]
```

Comment:
Explicit paths are strongly discouraged.

Is there a hard requirement for these to be explicit? If so, then we should ensure this is loudly documented as a dependency.

In general, though, the name of the library is typically sufficient and it will use the local system's LD_LIBRARY_PATH location(s). If the dependency isn't in one of those paths, the user is typically expected to add the necessary paths to that env var.